src/database/database.h

```diff
@@ -61,3 +63,25 @@ class PlayerDatabase
 	virtual bool removePlayer(const std::string &name) = 0;
 	virtual void listPlayers(std::vector<std::string> &res) = 0;
 };
+
+struct AuthEntry
+{
+	u64 id;
```

Comment:
this is just an implementation detail of the sqlite db, but I guess it can't be avoided to have it in here.

Reply:
It probably could, now that names are `UNIQUE`, but I've been asked to keep a numerical `id` anyway.

It may make a difference for future PostgreSQL implementations.